pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -73,6 +82,7 @@ def anonymise_dataset(  # pylint: disable = inconsistent-return-statements
     copy_dataset=True,
     replacement_strategy=None,
     identifying_keywords=None,
+    **_,
```

Comment:
presumably this prevents the remaining keys of the unpacked strategy dict/map from being exposed as variables/parameters.
I would document what this means because it's not a common usage.
A physicist who doesn't know what a dispatch table is will have difficulty understanding what **_  means, 
When I did a search, I got:
https://stackoverflow.com/questions/13726155/is-an-acceptable-way-to-ignore-arguments-in-python
and the approach (which was for *_ not **_) was considered arguable.
Beyond the "is it understandable to the average python programmer" (never mind physicist who isn't a programmer), there was also the issue that it would suppress errors if someone passed a named argument that wasn't going to be utilized.

Reply:
Ahh, I needed to make myself clearer. The portion of PyMedPhys that has its target audience as Medical Physicists and not programmers is all of the language that would end up appearing on https://docs.pymedphys.com/.

So docstrings on publicly exposed functions that have their text written onto the documentation site, need to be able to be understood by a medical physicist. The naming of internal functions, and the naming of internal APIs, I believe that's okay. What do you think?

That `**_` will go away however should I be explicit regarding your other comment.